index.html

```diff
@@ -0,0 +1,186 @@
+                 <li class="lang"><a href="#">javascript</a></li>
+               </ul>
+               <div>
+               <button type="submit" class="project-btn">See Project</button>
+               </div>
+         </div>
+     </section>
+     <section class="about-me">
+        <div class="about"> About <br>Myself
+            <p>I’m a software developer! I can help you <br> build a product, feature or website Look <br>
+              through some of my work and experience! If you like what you see and
+               have a project you need coded, don’t hestiate to contact me.
+            </p>
+
+            <a href="#" class="let-connect">LET'S CONNECT </a>
+
+            <ul class="icons-social-1">
+                <li><a class="icon icon-Github" href="#"> <img src="../icon-social/Github.png" alt="Github"/><a></a></li>
+                <li><a class="icon icon-Linkedin" href="#"> <img src="../icon-social/Linkedin.png" alt="Linkedin"></a></li>
+                <li><a class="icon icon-iOS" href="#"> <img src="../icon-social/iOS.png" alt="iOS"></a> </li>
+                <li><a class="icon icon-Twitter" href="#"><img src="../icon-social/Twitter.png" alt="Twitter"></a></li>
+                <li><a class="icon icon-Messenger" href="#"> <img src="../icon-social/Messenger.png" alt="Messenger"/></a></li>
+           </ul>
+          <div>
+          <button type="submit" class="project-btn">Get my resume</button>
+          </div>
+         <ul class="skill-list">
+            <li class="my-skill">
+              <div class="skill-container">
+                 <div class="skill-title">Languages</div>
+                 <button class="btn-chev" type="button">
+                    <img src="./images/Chevron - Down.png" alt="Dropdown-Arrow">
+                 </button>
+              </div>
+              <ul class="language-list">
+                <li><img class="c-width" src="./images/js.png" alt="JavaScript" />JavaScript</li>
+                <li><img class="c-width" src="./images/html5.png" alt="HTML" /> HTML</li>
+                <li><img class="c-width" src="./images/css3.png" alt="CSS" /> CSS</li>
+              </ul>
+            </li>
+            <li class="my-skill">
+              <div class="skill-container">
+                <div class="skill-title">Frameworks</div>
+                  <button class="btn-chev" type="button">
+                    <img src="./images/Chevron-right.png" alt="Dropdown-Right">
+                  </button>
+                </div>
+                <div class="line-1"><hr></div>
+            </li>
+            <li class="my-skill">
+              <div class="skill-container">
+                <div class="skill-title">Skills</div>
+                <button class="btn-chev" type="button">
+                    <img src="./images/Chevron-right.png" alt="Dropdown-Right">
+                </button>
+              </div>
+              <div class="line-1"><hr></div>
+            </li>
```

Comment:
:red_circle: You have done well adjusting the languages block according to the design template :+1: . Please make sure you align correctly the Frameworks and Skills items to match the Figma Design

Your Design
![still](https://user-images.githubusercontent.com/26736582/200058272-c1186682-daac-4f3f-a61c-fff553f8f0cd.png)

Figma Design

![fima](https://user-images.githubusercontent.com/26736582/200058332-7c2e83ce-b5e5-4756-9535-795f59fe3287.png)